src/frontend/components/Chat/StudentChat/index.tsx

```diff
@@ -6,11 +6,12 @@ import { ChatConversationDisplayer } from 'components/Chat/SharedChatComponents/
 import { InputBar } from 'components/Chat/SharedChatComponents/InputBar';
 import { InputDisplayNameOverlay } from 'components/Chat/SharedChatComponents/InputDisplayNameOverlay';
 import { JoinChatButton } from 'components/Chat/SharedChatComponents/JoinChatButton';
+import { useChatItemState } from 'data/stores/useChatItemsStore';
 import { converse } from 'utils/window';
 
 export const StudentChat = () => {
+  const { displayName } = useChatItemState.getState();
```

Comment:
Same as before, you should not use the store like this, you are not using it like a hook.